### PR TITLE
Add PJS to Builder and Pachctl Misc gRPC

### PR DESCRIPTION
**Pull request by FahadBSyed**

### CODE REVIEW

File: src/internal/pachd/builder.go [hunk 1]
```diff
@@ -283,6 +286,22 @@ func (b *builder) registerPFSServer(ctx context.Context) error {
 	return nil
 }
 
+func (b *builder) registerPJSServer(ctx context.Context) error {
+	apiServer := pjs_server.NewAPIServer(pjs_server.Env{
+		DB:               b.env.GetDBClient(),
+		GetPermissionser: b.env.AuthServer(),
+		GetAuthToken:     auth.GetAuthToken,
+		// TODO: figure out how to set this.
+		GetStorageClient: func(ctx context.Context) storage.FilesetClient {
```
Comment (brendoncarroll) [inline src/internal/pachd/builder.go:295]: Why do we need a callback here?  This can just be `storage.FilesetClient`.

**Review by brendoncarroll — APPROVED** [review]

File: src/internal/pachd/builder.go (hunk 1 above)
Comment (jrockway) [reply src/internal/pachd/builder.go:295]: Agree.  Many clients close over ctx, but fortunately not the FilesetClient.

Comment (jrockway) [reply src/internal/pachd/builder.go:295]: Though since the client doesn't have a conn this early, you probably have to do it like this.

Comment (FahadBSyed) [reply src/internal/pachd/builder.go:295]: Bryce's PR will actually change this to a StorageServer, so when that gets merged in, he or I will have to update this anyways 😆

**Pull request merged**